data-pipeline-flink/de-normalization/src/main/resources/de-normalization.conf

```diff
@@ -33,4 +33,7 @@ redis-meta {
     contentstore.id = 5
     dialcodestore.id = 6
   }
-}
\ No newline at end of file
+}
+
+//config version
+version = v2
```

Comment:
since this is user specific - rename the variable as user_denorm_version

Reply:
Done